packages/app/src/cli/services/app-logs/write-app-logs.test.ts

```diff
@@ -42,6 +43,25 @@ describe('writeAppLogsToFile', () => {
     expect(writeLog).toHaveBeenCalledWith(expect.stringContaining(path), logData)
   })
 
+  test('calls writeLog with the right data with log_type !== undefined', async () => {
+    // Given
+    const appLog = {
+      ...APP_LOG,
+      log_type: 'function_run',
+    }
+    const logData = expectedLogDataFromAppEvent(appLog)
+
+    // determine the fileName and path
+    const fileName = `20240522_150641_827Z_${APP_LOG.source_namespace}_${APP_LOG.source}`
+    const path = joinPath(API_KEY, fileName)
+
+    const returnedPath = await writeAppLogsToFile({appLog, apiKey: API_KEY, stdout})
```

Comment:
Super nitpick: I think the CLI team tries to annotate given/when/then

```suggestion
    // When
    const returnedPath = await writeAppLogsToFile({appLog, apiKey: API_KEY, stdout})
```